### PR TITLE
[Feature #739] Add plugin-spark-sink-kafka

### PR DESCRIPTION
**Pull request by simon824**
## Purpose of this pull request
#739 Add plugin-spark-sink-kafka

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] Change does not need document change, or I will submit document change to https://github.com/InterestingLab/seatunnel-docs later

### CODE REVIEW

**Comment by simon824** [comment]
cc @RickyHuo @CalvinKirs

File: plugin-spark-sink-kafka/pom.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
+            </plugin>
+
+            <plugin>
+                <groupId>net.alchim31.maven</groupId>
+                <artifactId>scala-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
```
Comment (CalvinKirs) [inline plugin-spark-sink-kafka/pom.xml:84]: hi, here is no longer needed, can you update the upstream code

Comment (simon824) [reply plugin-spark-sink-kafka/pom.xml]: Got it.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**